### ISSUE TITLE
[FR] Cut out circle on the trigger button. Make it respond to touch like haptic feedback.

### ISSUE THREAD

**Issue by HarryHeights** [tacticalDevC/PanicTrigger#9]
**Is your feature request related to a problem? Please describe.**
This isn't quite a problem. I like that the full screen is a trigger. This would come in handy in serious emergency cases. However, it still wouldn't be bad to cut out a circle around the trigger button, because subconsciously my mind is moved to hit only the trigger text not knowing other parts of the screen works as a trigger.
Then secondly, a haptic feedback would be great. When I tapped the screen, it didn't feel like I pressed anything. I thought the app didn't work, not until I got an sms on my other phone.

This part is unrelated to my feature request. I wanted to ask how you deal with user privacy when the app is used. 
Do you have a privacy policy statement? It would be a great addition to the app. Especially for users who are skeptical about their privacy policy. If you don't a policy statement yet, I could help you create it.

**Comment by tacticalDevC** [comment]
Hello friend
Thanks for this feature request. I appreciate your help :)

First I'd like to talk about the actual FR. I see your point of view and it makes sense to give the trigger button a bit margin.
Next point is a nice enhancement too. As the dev it was clear to me that the button works, not thinking about other people which simply don't know that there has been any kind of action. Thanks for that too.

Your question is reasonable and you have the right to know what happens with your data. First of all. I don't collect any kind of data. I have absolutely no clue who is using my app. So what happens on your phone stays on your phone. Next question would be "what happens with my data if I tap the trigger"? Umm well at this time of development your position is automatically collected and sent to your preconfigured contacts. Who you share your location with is  none of my business. There'll be an option to turn location gathering off. 

I don't have a privacy statement and I would be really grateful if you could help me creating one.

One question. Do you test my app or use it for eventual emergencies? If the second matches you I would like to know how you use it.

Cheers from Austria :)

**Comment by HarryHeights** [comment]
> Do you test my app or use it for eventual emergencies?

Well, I downloaded the app not just to test but use it. I will be honest to you. My country isn't a safe place. Everything could be normal and the next second there's chaos. So the app was downloaded for use.

> I would like to know how you use it.

Well, I set it up as any other user would. I added my parents and close friends contact. Entered the texts and all those stuff. My parents don't use smartphones, which means they can't install the app. They'd only get the text and not automatically call back.

Btw, now I just thought of it, is it possible for you to integrate the app with a map?. Like it tracks my current location and when I click the trigger button, it sends the text alongside my location. It'd be a great advantage. What do you think?

**Comment by tacticalDevC** [comment]
Thank you for your feedback :) I really appreciate it.

Your thoughts are right. It is a great advantage but I already implemented that with OSM. If you get the panic message and the alarm goes off you can click on the notification and will get a map with the current location. Although this feature is still a little bit buggy and not documented it is there.

**Comment by HarryHeights** [comment]
Oh okay. It means that, the recipient phone has to be a smartphone. Because I test the app's effectiveness with my other phone. Really small phone , just for calls. So I get only the text and some weird numbers.

**Comment by tacticalDevC** [comment]
That's right. The other phone needs to be a smartphone

Those weird numbers are your coordinates.

**Comment by HarryHeights** [comment]
I thought as much. Is there no way the app can extract my location as a text and send to a recipient?

**Comment by tacticalDevC** [comment]
You mean like an address?

**Comment by HarryHeights** [comment]
Like a general location where I'm at, at that moment.

**Comment by tacticalDevC** [comment]
As far as I understand you mean coordinates?

**Comment by HarryHeights** [comment]
Yea maybe. Okay, I'm totally confused. But, lemme rephrase what I was thinking again. Now you said the app sends coordinates of my location when I send a panic message. I was asking if it would be possible for the app to fetch my location and send it as text instead of co-ordinates.

For example, I'm in at a location lemme say X. I send a panic message and the app sends the coordinates   of my location X. My suggestion is instead of sending my coordinates, let the app specifically tell the user I sent the panic message to that I'm at X.

**Comment by tacticalDevC** [comment]
I'm really sorry I still don't understand. Maybe because my English is not the best. Could you explain your suggestion for idiots?

**Comment by HarryHeights** [comment]
I'm sorry, I thought you were an Englishman. My bad.
When I hit the trigger button, I still got same message as I did when I sent the message to a normal phone, on my smartphone. I got the panic message and the coordinates of my current location.

Someone who gets my message may not know what those numbers are. So I was asking;

Is there a way my location is actually spelt out to the recipient instead of my coordinates?

**Comment by tacticalDevC** [comment]
I think I understand. You want a sms like that:

Panic
Latitude: <some number>
Longitude: <some number>

Please correct me if I'm not right

**Comment by HarryHeights** [comment]
Nope I want an SMS 

Panic!! at Lowton avenue.

**Comment by tacticalDevC** [comment]
Okay now I understand xD you want an resolved location

**Comment by HarryHeights** [comment]
Exactly!

**Comment by tacticalDevC** [comment]
Closing this issue since the main topic is implemented. Opening for the location-resolving-issue a new issue.

**Comment by HarryHeights** [comment]
Okay great.
